packages/client/hmi-client/src/services/concept.ts

```diff
@@ -210,9 +211,33 @@ const autoEntityMapping = async (
 				});
 			}
 		});
+
+		// Match by string if no target is found
+		if (isEmpty(currentTargetId)) {
+			targetEntities.some((target) => {
+				if (target.id.startsWith(distinctSourceId) || distinctSourceId.startsWith(target.id)) {
```

Comment:
Why use `startsWith` for string comparison? 
What happens when you compare `infected` and `infected_old` both "start with" infected but do not match?

Is `stringOne.toLower() === stringTwo.toLower()` not better?

Reply:
I considered that but take a look at this, the caps helps here:
![image](https://github.com/DARPA-ASKEM/terarium/assets/28237258/a533a446-c4eb-4ef1-9ffc-08df3eda439a)